Plugins/NativeEngine/Source/NativeEngine.h

```diff
@@ -42,6 +42,9 @@ namespace Babylon
         uint32_t Width{0};
         uint32_t Height{0};
         uint32_t Flags{0};
+        // CreationFlags contains flags used at texture creation
+        // regarding BLIT support and READBACK
+        uint64_t CreationFlags{0};
```

Comment:
Why is this new member needed? The logic looks identical, does Flags get overwritten at some point or something?

Reply:
`Flags` is used to texture properties when used for rendering (sampling mode, wrapping, anisotropic, ...) and changed by JS.
`CreationFlags` is the caps for the texture: will be used for blit, readback, is RT,... and should only change if underlying bgfx texture is destroyed and recreated.